exercises/anagram/.meta/gen.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"text/template"
+
+	"github.com/exercism/xgo/gen"
```

Comment:
The import path used in other generators for referencing the gen package is:

>       "../../../gen"

Will remove the need to have github.com/ ... in the xgo tree/workspace.